repos/system_upgrade/el8toel9/actors/nischeck/actor.py

```diff
@@ -0,0 +1,19 @@
+from leapp.actors import Actor
+from leapp.libraries.actor.nischeck import report_nis
+from leapp.models import InstalledRedHatSignedRPM, NISConfig, Report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+
+class NISCheck(Actor):
+    """
+    Checks if any of NIS components is installed and configured
+    on the system and warns users about discontinuation.
+    """
+
+    name = 'nis_check'
+    consumes = (InstalledRedHatSignedRPM, NISConfig,)
```

Comment:
Here's an unnecessary trailing comma
```suggestion
    consumes = (InstalledRedHatSignedRPM, NISConfig)
```
If there's more than one entry it is already a tuple, a trailing comma is only needed when there's only one item, to indicate that it is a tuple and not a constrained expression

Reply:
Changed.